### DISCUSSION TITLE
GCP Cloud SQL support

### DISCUSSION THREAD

**Discussion by NextBasket-Petyo** [Releem/releem-agent#259] [Feature Requests]
Hello I see Amazon RDS is supported, it would be great if we can have GCP Cloud SQL support as well.

**Comment by drupaladmin** [comment]
Thanks for your suggestion. Converting to feature request.

**Comment by drupaladmin** [comment]
Added https://github.com/Releem/mysqlconfigurer/releases/tag/1.22.0.4

**Comment by drupaladmin** [comment]
Hey @NextBasket-Petyo , we've added GCP Cloud SQL support. 
Here is the installation guide: https://docs.releem.com/releem-agent/installation-guides/cloud-managed-gcp-cloud-sql-automatic-installation
